tests/utils/test_readers_safedir.py

```
@@ -0,0 +1,1443 @@
+            read_ebook_file,
+            read_hdf5_file,
+            read_netcdf_file,
+            read_mat_file,
+            read_dxf_file,
+            read_dwg_file,
+            read_step_file,
+            read_iges_file,
+        ],
+    )
+    def test_rejects_both_args_missing(self, reader) -> None:  # type: ignore[no-untyped-def]
+        with pytest.raises(ValueError):
```

Ruff (0.15.13)

[warning] `pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)